### PR TITLE
Add installation of library using github release

### PR DESCRIPTION
**Pull request by jgoldfar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,9 @@
 GUI building, using the `libui` library: [https://github.com/andlabs/libui](https://github.com/andlabs/libui)
 <br/>
 
-[![Build Status](https://travis-ci.org/joa-quim/Libui.jl.svg?branch=master)](https://travis-ci.org/joa-quim/Libui.jl)
+[![Build Status](https://travis-ci.org/jgoldfar/Libui.jl.svg?branch=master)](https://travis-ci.org/jgoldfar/Libui.jl)
+[![Coverage Status](https://coveralls.io/repos/github/jgoldfar/Libui.jl/badge.svg?branch=master)](https://coveralls.io/github/jgoldfar/Libui.jl?branch=master)
+[![codecov.io](http://codecov.io/github/jgoldfar/Libui.jl/coverage.svg?branch=master)](http://c    odecov.io/github/jgoldfar/Libui.jl?branch=master)
```
Comment (waldyrious) [inline README.md]: There's some extra whitespace in the URL:

    c    odecov.io

Probably an accidental press of the tab key?

**Comment by jgoldfar** [comment]
Comment addressed; accidentally increased the scope of this branch a bit...

**Comment by joa-quim** [comment]
Thanks, I'll merge this PR but please notice that I don't like indentations of two spaces so I'll restore a indentation with tabs. But this is detail.

**Comment by jgoldfar** [comment]
Thanks! Not sure what editor setting led to that indentation level...

**Pull request merged**